feature/results/src/main/java/com/android/developers/androidify/customize/xr/CustomizeExportScreenSpatial.kt

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright 2025 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.android.developers.androidify.customize.xr
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.size
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.SnackbarHostState
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.dp
+import androidx.xr.compose.spatial.ContentEdge
+import androidx.xr.compose.spatial.Orbiter
+import androidx.xr.compose.spatial.OrbiterOffsetType
+import androidx.xr.compose.subspace.SpatialBox
+import androidx.xr.compose.subspace.SpatialColumn
+import androidx.xr.compose.subspace.SpatialLayoutSpacer
+import androidx.xr.compose.subspace.SpatialPanel
+import androidx.xr.compose.subspace.SpatialRow
+import androidx.xr.compose.subspace.layout.SubspaceModifier
+import androidx.xr.compose.subspace.layout.fillMaxHeight
+import androidx.xr.compose.subspace.layout.fillMaxWidth
+import androidx.xr.compose.subspace.layout.offset
+import androidx.xr.compose.subspace.layout.width
+import com.android.developers.androidify.customize.CustomizeExportScreenScaffold
+import com.android.developers.androidify.customize.CustomizeExportState
+import com.android.developers.androidify.customize.ExportImageCanvas
+import com.android.developers.androidify.xr.DisableSharedTransition
+import com.android.developers.androidify.xr.MainPanelWorkaround
+import com.android.developers.androidify.xr.SquiggleBackgroundSubspace
+
+@Composable
+fun CustomizeExportLayoutSpatial(
+    state: CustomizeExportState,
+    snackbarHostState: SnackbarHostState,
+    imageResult: @Composable (ExportImageCanvas.() -> Unit),
+    toolDetail: @Composable (Modifier, Boolean) -> Unit,
```

Comment:
It'd be nice if you could elaborate on the Boolean here that's passed around, is it the selected state Boolean? Just reading the code here and it's not entirely clear from first glance. Maybe it'd make sense to have an interface to represent this set of parameters instead?